src/ElectronApp.ts

```diff
@@ -30,6 +30,23 @@ export default class ElectronApp {
     this.autoUpdater = new AutoUpdater()
     this.protocol = process.env.NODE_ENV === 'development' ? DEEP_LINK_PROTOCOL_DEV : DEEP_LINK_PROTOCOL
 
+    const template = [
+      {
+        label: 'Help',
+        submenu: [
+          {
+            label: 'Learn More',
+            click() {
+              electron.shell.openExternal('https://support.remote.it/hc/en-us')
```

Comment:
Could you check with @bstrech and see if there is a short.io link we should use? Also is there a way to keep the other default menus? They seem to be lost.